argocd/datadog_checks/argocd/metrics.py

```diff
@@ -75,6 +75,13 @@
     'argocd_repo_pending_request_total': 'repo.pending.request.total',
 }
 
+# https://argo-cd.readthedocs.io/en/stable/operator-manual/notifications/monitoring/
+NOTIFICATIONS_CONTROLLER = {
+    'argocd_notifications_deliveries': 'notifications.deliveries.total',
+    'argocd_notifications_trigger_eval': 'notifications.trigger_eval.total',
```

Comment:
Nit:

```suggestion
    'argocd_notifications_deliveries': 'notifications.deliveries',
    'argocd_notifications_trigger_eval': 'notifications.trigger_eval',
```

`argocd.notifications_controller.notifications.trigger_eval.count` might be a better representation here. But it's non blocking.